bears/python/ShebangBear.py

```diff
@@ -0,0 +1,34 @@
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class ShebangBear(LocalBear):
+    LANGUAGES = {'Python', 'Python 2', 'Python 3'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Syntax'}
+
+    def run(self, filename, file):
+        """
+            Ensure that the file begins with the Shebang operator.
+        """
+        if len(file) == 0 or '#!/usr/bin/env' in file[0]:
+            return
+
+        diff = Diff(file)
+        mes = ''
+
+        if '#!' in file[0]:
```

Comment:
@yukiisbored so first of all, the bear should be moved from the python bears directory.
Now, let's discuss the case where the user uses something like `#!/usr/bin/python` , how can we check that the user doesn't  use the general form of the shebang operator and then deduce that script is written in python(or any other language) ?
Do you think that the current check can be improved somehow? 
Thanks 😃